### PR TITLE
#440 rendercache

### PR DESCRIPTION
**Pull request by rsupnekar**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by rsupnekar** [comment]
Includes changes made to implement overlay.apply and support overlay.addFeatures for existing features.

File: mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java [hunk 1]
```diff
@@ -85,6 +100,22 @@ public void resetDirty() {
         this.setDirty(false);
     }
 
+    public void setHighPlacemarkAttributes(final PlacemarkAttributes pma) {
+        highPlacemarkAttributes = pma;
```
Comment (jrummel) [inline mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java]: Could preface these with this

File: mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java [hunk 2]
```diff
@@ -106,20 +137,20 @@ public void updateSymbol(MilStdSymbol symbol) {
...
-            this.setDirty(true);
-            this.oModifiers = oMod;
-        }
-
-        if (!this.oAttributes.toString().equals(oAttr.toString())) {
-            // The attributes has changed, mark dirty.
-            this.setDirty(true);
-            this.oAttributes = oAttr;
-        }
+//        SparseArray oMod = this.oRenderer.getUnitModifiers(this.getMapInstance(), this.getFeature());
```
Comment (jrummel) [inline mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java]: Comment as to why this is being commented out

**Review by jrummel — APPROVED** [review]

File: mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java (hunk 2 above)
Comment (rsupnekar) [reply mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/feature/MilStd2525SinglePoint.java]: good catch, that was a test that shouldn't be commented

**Pull request merged**